eZ/Publish/API/Repository/Tests/SetupFactory/LegacySolr.php

```diff
@@ -16,6 +16,8 @@
 use PDO;
 use RuntimeException;
 use eZ\Publish\API\Repository\Tests\SearchServiceTranslationLanguageFallbackTest;
+use Symfony\Component\DependencyInjection\Loader\YamlFileLoader;
+use Symfony\Component\Config\FileLocator;
```

Comment:
Moved in https://github.com/ezsystems/ezpublish-kernel/commit/d4d1a0be006f1087801a46527225f6b0034062c9 and https://github.com/ezsystems/ezplatform-solr-search-engine/commit/1c771fbec825b4bb9ef84790b90fdb09326b3236.